pycyphal/dsdl/_import_hook.py

```diff
@@ -101,7 +101,9 @@ def find_spec(
 
 
 def get_default_lookup_dirs() -> Sequence[str]:
-    return os.environ.get("CYPHAL_PATH", "").replace(os.pathsep, ";").split(";")
+    dirs = os.environ.get("CYPHAL_PATH", "").replace(os.pathsep, ";").split(";")
+    dirs = [dir for dir in dirs if dir]  # filter out empty strings
```

Comment:
1. Add `.strip()` to also ignore whitespace-only strings.
2. `dir` is a [built-in entity](https://docs.python.org/3/library/functions.html#dir) in Python and the linter will (or should) complain about this.

```suggestion
    dirs = [d for d in dirs if d.strip()]  # filter out empty strings
```